hydromt_wflow/wflow.py

```diff
@@ -3684,7 +3700,6 @@ def basins(self):
                 .set_index("value")
                 .sort_index()
             )
-            gdf.index.name = self._MAPS["basins"]
```

Comment:
Why remove this line? This would change the name of the index column from wflow_subcatch to value. Apparently we don't use this in other places in the code but why change?

Reply:
It broke the tests

Reply:
Which test did it break? Maybe actually using `wflow_subcatch` rather than `value` as name is better no? While we are breaking behavior of self.basins anyhow :)